test/distributed/org/apache/cassandra/distributed/upgrade/DropCompactStorageTest.java

```diff
@@ -35,7 +35,7 @@ public void testDropCompactStorage() throws Throwable
         new TestCase()
         .nodes(2)
         .nodesToUpgrade(1, 2)
-        .upgradesToCurrentFrom(v30)
+        .upgradesToCurrentFrom(OLDEST)
         .withConfig(config -> config.with(GOSSIP, NETWORK).set("enable_drop_compact_storage", true))
         .setup((cluster) -> {
             cluster.schemaChange("CREATE TABLE " + KEYSPACE + ".tbl (pk int, ck int, PRIMARY KEY (pk, ck)) WITH COMPACT STORAGE");
```

Comment:
is it still possible to `CREATE TABLE WITH COMPACT STORAGE` in 4.0+  ? 

ref: https://github.com/apache/cassandra/pull/3782#issuecomment-2578362286

Reply:
https://github.com/apache/cassandra/pull/3782/files/d4cdc3ab58c86f79c7ddeb18ba95ebbd4aa487d5#r1907840114